### PR TITLE
feat: generated mainnet-mirror genesis

### PR DESCRIPTION
**Pull request by daniel1302**

### CODE REVIEW

File: mainnet-mirror/templates/genesis-template.json [hunk 1]
```diff
@@ -44,112 +43,127 @@
...
+            "governance.proposal.updateAsset.minVoterBalance": "1000000000000000000",
+            "governance.proposal.updateAsset.requiredMajority": "0.66",
+            "governance.proposal.updateAsset.requiredParticipation": "0.10",
             "governance.proposal.updateMarket.maxClose": "720h",
             "governance.proposal.updateMarket.maxEnact": "720h",
             "governance.proposal.updateMarket.minClose": "10m",
             "governance.proposal.updateMarket.minEnact": "10m",
-            "governance.proposal.updateMarket.minProposerBalance": "20000000000000000000000",
+            "governance.proposal.updateMarket.minProposerBalance": "2000000000000000000",
+            "governance.proposal.updateMarket.minProposerEquityLikeShare": "0.05",
```
Comment (davidsiska-vega) [inline mainnet-mirror/templates/genesis-template.json:81]:
```suggestion
            "governance.proposal.updateMarket.minProposerEquityLikeShare": "0.0",
```

File: mainnet-mirror/templates/genesis-template.json [hunk 2]
```diff
@@ -44,112 +43,127 @@
...
             "governance.proposal.updateMarket.minVoterBalance": "1000000000000000000",
             "governance.proposal.updateMarket.requiredMajority": "0.66",
+            "governance.proposal.updateMarket.requiredMajorityLP": "0.66",
             "governance.proposal.updateMarket.requiredParticipation": "0.20",
+            "governance.proposal.updateMarket.requiredParticipationLP": "0.00001",
```
Comment (davidsiska-vega) [inline mainnet-mirror/templates/genesis-template.json:86]:
```suggestion
            "governance.proposal.updateMarket.requiredParticipationLP": "0.0",
```

File: mainnet-mirror/genesis.json [hunk 3]
```diff
@@ -290,7 +303,7 @@
     },
     "version": {}
   },
-  "genesis_time": "2023-01-03T18:56:26.31098759Z",
+  "genesis_time": "2023-01-06T14:10:51.279209Z",
   "initial_height": "0",
   "validators": [
```
Comment (davidsiska-vega) [inline mainnet-mirror/genesis.json:308]: @daniel1302 are these the validator identities we want here? I guess yes, if that's what the mainnet mirror is running with but no if it's the mainnet identities.

Comment (daniel1302) [reply mainnet-mirror/genesis.json:308]: This is the mainnet mirror. We will have to update mainnet genesis with similar PR, here: https://github.com/vegaprotocol/networks/blob/master/mainnet1/genesis.json

**Pull request merged**